entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,184 @@
+
+## Download and run the sample project
+
+To run and explore this sample application, download the code on GitHub.
+
+Once downloaded, open the solution file `EFGettingStarted.sln` in Visual Studio or Visual Studio for Mac and run the application on the platform of your choice.
+
+When the app first starts, it will populate the local SQLite database with 2 entries representing blogs.
+
+Click the **Add** button in the toolbar.
+
+A new page will appear that allows you to enter information about a new blog. 
+
+Fill out all the info and click **Save** from the toolbar. The new blog will save to the app's SQLite database and will show in the list.
+
+You can click on one of the blog entries in the list and see any posts for that blog.
+
+Click **Add** in the toolbar.
+
+A page then appears that allows you to fill out information about a new blog post. Fill out all the information and click **Save** in the toolbar.
+
+The new post will be associated to the blog post you clicked on in a previous step and will save to the app's SQLite database and show in the list.
+
+Go back to the blog list page. And click **Delete All** in the toolbar. All blogs and their corresponding posts will then be deleted from the app's SQLite database.
+
+## Explore the code
+
+The following sections will walk you through the code in the sample project that reads, creates, updates, and deletes data from a SQLite database using EF Core with Xamarin.Forms.
+
+It is assumed that you are familiar with the Xamarin.Forms topics of displaying data and navigating between pages.
+
+## Entity Framework Core NuGet packages
+
+To create Xamarin.Forms apps with EF Core, you install the package for the EF Core database provider(s) you want to target into all of the projects in the Xamarin.Forms solution. This tutorial uses SQLite.
+
+The following NuGet packages are needed in each of the projects in the Xamarin.Forms solution.
+
+* `Microsoft.EntityFrameworkCore`
+* `Microsoft.EntityFrameworkCore.Sqlite`
+
+## Model classes
+
+Each table in the SQLite database accessed through EF Core is modeled in a class. In this sample 2 classes are used: `Blog` and `Post` which can be found in the `Models` folder.
+
+The model classes are composed only of properties, which model columns in the database.
+
+* **Blog.cs**
+
+  [!code-csharp[Main](../../../samples/core/Xamarin/EFGetStarted/Models/Blog.cs)]
+
+* The `Posts` property defines a parent-child relationship between `Blog` and `Post`.
+
+* **Post.cs**
+
+  [!code-csharp[Main](../../../samples/core/Xamarin/EFGetStarted/Models/Post.cs)]
+
+* The `BlogId` and `Blog` properties relate back to the parent `Blog` object for the instance of the `Post`.
+
+## Data context
+
+The `BloggingContext` class is located in the `Services` folder and inherits from the EF Core `DbContext` class. A `DbContext` is used to group together database queries and changes.
+
+  [!code-csharp[Main](../../../samples/core/Xamarin/EFGetStarted/Services/BloggingContext.cs)
```

Comment:
This isn't showing up correctly in the preview... I think you're missing the right ]